scripts/install/lazylibrarian.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+# LazyLibrarian install script for swizzin
+# Author: Aethaeran
+
+##########################################################################
+# References
+##########################################################################
+
+# https://github.com/Aethaeran/swizzin/blob/feat/jdownloader/scripts/install/jdownloader.sh
+# https://lazylibrarian.gitlab.io/install/
+# https://www.reddit.com/r/LazyLibrarian/comments/jw5e22/lazylibrarian_calibre_calibreweb_booksonic/
+# https://github.com/lazylibrarian/LazyLibrarian
+# https://gitlab.com/LazyLibrarian/LazyLibrarian
+# https://lazylibrarian.gitlab.io/config_commandline/
+# https://github.com/swizzin/swizzin/pull/743
+# https://lazylibrarian.gitlab.io/config_users/
+# https://lazylibrarian.gitlab.io/config_downloaders/
+# https://www.reddit.com/r/LazyLibrarian/
+# https://lazylibrarian.gitlab.io/api/
+
+##########################################################################
+# Import Sources
+##########################################################################
+
+. /etc/swizzin/sources/functions/color_echo
+. /etc/swizzin/sources/functions/pyenv
+. /etc/swizzin/sources/functions/users
+. /etc/swizzin/sources/functions/utils
+
+##########################################################################
+# Variables
+##########################################################################
+
+app_name="lazylibrarian"
+pretty_name="LazyLibrarian"
+default_port="5299"
+master=$(_get_master_username)
```

Comment:
I would suggest making this an environment variable, you can check #719 how that looks. You just need to paste the "logic" for retrieving that into the master variable. Might want tot rename that to "user" as well.

Reply:
I like this idea because then people could optionally install it to a different user using the variable. Smart cookie.

Reply:
should be done now